### ISSUE TITLE
Formula result #VALUE! instead of 0

### ISSUE THREAD

**Issue by Aurelianus** [EPPlusSoftware/EPPlus#1225]
Hi,

with version 7 the following is not working, which worked in version 6 and in MsExcel:

  
```
[TestMethod]
public void Formula()
{
    using (var package = new ExcelPackage())
    {
        var ws = package.Workbook.Worksheets.Add("Test");

        ws.Cells["B1"].Formula = "-E1";
        ws.Cells["M1"].Value = "RAEP";
        ws.Cells["D1"].Value = "PV";
        ws.Cells["G1"].Formula = "IF(AND(M1=\"RAEP\",D1=\"PV\"),B1+E1,B1)";
        ws.Calculate();

        Assert.AreEqual(0d, ws.Cells["G1"].Value);

        ws.Dispose();
    }
}
```

Version 7 result is #VALUE! where Version 6 returns 0 as well as MsExcel.

**Comment by JanKallman** [comment]
Thanks for reporting this. The #value! error happens when you negate an empty cell. I will provide a fix for this bug.

**Comment by Aurelianus** [comment]
Thanks for your quick response. :thumbsup:

**Comment by JanKallman** [comment]
Fixed in EPPlus 7.0.5
